hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/delete/job/DeleteExpungeProcessor.java

```diff
@@ -66,7 +67,12 @@
 		List<String> retval = new ArrayList<>();
 
 		String pidListString = thePids.toString().replace("[", "(").replace("]", ")");
```

Comment:
I'm uncomfortable with this line.  `List` does not define toString(), we assume a particular format.
I'd feel safer with something explicit.  Don't we have a helper somewhere?
```suggestion
		String pidListString = "(" + ",".join(thePids) +  ")";
```

Reply:
I'm right there with you. This was pre-existing, happy to change it.